kubectl-plugin/README.md

```diff
@@ -18,22 +18,79 @@ The general command structure is `kubectl mayastor <operation> <resource>` where
 
 The plugin needs to be able to connect to the REST server in order to make the appropriate REST calls. The IP address and port number of the REST server can be provided through the use of the `--rest` command line argument. If the `--rest` argument is omitted, the plugin will attempt to make use of the kubeconfig file to determine the IP of the master node of the cluster. Should the kubeconfig file contain multiple clusters, then the first cluster will be selected.
 
-### Examples
 
-#### Volumes
-Getting all volumes:\
-`kubectl mayastor get volumes`
+### Examples and Outputs
 
-Getting a specific volume:\
-`kubectl mayastor get volume ec4e66fd-3b33-4439-b504-d49aba53da26`
+1. List Volumes
+```
+❯ kubectl mayastor get volumes
+ ID                                    PATHS  REPLICAS  PROTOCOL  STATUS  SIZE 
+ 18e30e83-b106-4e0d-9fb6-2b04e761e18a  1      4         none      Online  10485761 
+ 0c08667c-8b59-4d11-9192-b54e27e0ce0f  1      4         none      Online  10485761 
 
-Scaling a specific volume:\
-`kubectl mayastor scale volume ec4e66fd-3b33-4439-b504-d49aba53da26 2`
+``` 
+2. Get Volume by ID
+```
+❯ kubectl mayastor get volume 18e30e83-b106-4e0d-9fb6-2b04e761e18a
+ ID                                    PATHS  REPLICAS  PROTOCOL  STATUS  SIZE 
+ 18e30e83-b106-4e0d-9fb6-2b04e761e18a  1      4         none      Online  10485761 
 
-#### Pools
+```
+3. List Pools
```

Comment:
I think we want to change `List` to `Get` because the operation is now a `Get` operation.